Test/JenkinsWebApiUnitTestFramework461/JenkinsWebApiUnitTestFramework461.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>JenkinsWebApiUnitTestFramework461</RootNamespace>
```

Comment:
likely JenkinsWebApiUnitTestFramework461 should be renamed to JenkinsWebApiUnitTestFramework462